packages/matchbox/src/helpers/math.js

```diff
@@ -0,0 +1,16 @@
+export function roundToPlaces(number, places) {
+  const multiplier = Math.pow(10, places);
+  return Math.round(number * multiplier) / multiplier;
+}
+
+export function trim(n, lower, upper) {
```

Comment:
Nomenclature nitpick: I believe this is traditionally known as `clamp()` to distinguish it from the various string trim utilities.